test/unittest/layers/layers_common_tests.h

```diff
@@ -38,7 +38,7 @@ using LayerSemanticsParamType =
              std::string /** Type of Layer */,
              std::vector<std::string> /** Valid Properties */,
              std::vector<std::string> /** Invalid Properties */,
```

Comment:
As per offline discussion yesterday, how about we delete invalid properties here?

Reply:
done.